new_src/main.py

```
@@ -64,5 +68,79 @@ def run_cli():
+            print(f"'chmod +x {script_name}' 명령어를 실행하여 권한을 부여해주세요.")
+            sys.exit(1)
+
+    print("=" * 40)
+    print(f"'{script_name}' 스크립트를 실행하여 웹 서비스를 시작합니다...")
+    print("=" * 40)
+    
+    try:
+        # subprocess.run을 사용하여 외부 쉘 스크립트를 실행합니다.
+        # check=True: 스크립트 실행이 실패하면 예외를 발생시킵니다.
+        # 이 스크립트가 백그라운드(&)로 프로세스를 띄우므로, 파이썬은 바로 다음 코드를 실행합니다.
+        _ = subprocess.run([script_path], check=True, text=True)
```

Ruff (0.14.3)

`subprocess` call: check for execution of untrusted input

(S603)